wishlist.toml

```diff
@@ -1547,6 +1547,12 @@ description = "An efficient ActivityPub Server, for small installs with multiple
 upstream = "https://github.com/jointakahe/takahe"
 website = "https://jointakahe.org"
 
+[tasks-md]
+name = "Tasks.md"
+description = "App for managing tasks lists with markdown support."
+upstream = "https://github.com/BaldissaraMatheus/Tasks.md"
```

Comment:
Will be nice to transfer the app to YunoHost-app repo

Reply:
:grimacing: but that's the upstream :grimacing:

Reply:
I'm so busy right now with work, I shouldn't try to contribute 😅